_studio/mfx_lib/cmrt_cross_platform/src/cmrt_cross_platform.cpp

```diff
@@ -23,12 +23,8 @@
 #include "vm_shared_object.h"
 #include "cmrt_cross_platform.h"
 
-#if defined(LINUX64)
-const vm_char * DLL_NAME_LINUX = VM_STRING("igfxcmrt64.so");
-#elif defined(LINUX32)
-const vm_char * DLL_NAME_LINUX = VM_STRING("igfxcmrt32.so");
-#else
-#error "undefined configuration"
+#if defined(LINUX64) || defined(LINUX32)
+const vm_char * DLL_NAME_LINUX = VM_STRING("libigfxcmrt.so");
```

Comment:
No, this is incorrect. That's a linker name which will not be available in runtime packages, we need to load by soname: "libigfxcmrt.so.7" where 7 is a major API version of cmrt: https://github.com/intel/media-driver/blob/master/cmrtlib/agnostic/share/cm_rt.h#L77